site/manifest-config.js

```diff
@@ -1,11 +1,32 @@
 const colors = require('./colors');
 
 module.exports = {
-  name: 'Mate Gatsby Starter',
-  short_name: 'Mate Gatsby Starter',
+  name: 'Reach4Help',
+  short_name: 'Reach4Help',
   start_url: '/',
   background_color: colors.background,
   theme_color: colors.primary,
   display: 'minimal-ui',
-  icon: 'media/icon.png',
+  icons: [
```

Comment:
If I am not mistaken out of the box this already generates all icon sizes, unless you need to override it that I think it's not the case.

We just need the bigger icon and gatsby manifest takes care of the rest
https://www.gatsbyjs.org/packages/gatsby-plugin-manifest/

![image](https://user-images.githubusercontent.com/9373787/77656053-36375080-6f6b-11ea-9ac4-e09213445430.png)

Reply:
Oh that's great, yep that works!